main.go

```diff
@@ -110,20 +104,33 @@ func buildRsync(rs *Rsync) *exec.Cmd {
 		args = append(args, fmt.Sprintf("--exclude=%s", rs.Exclude))
 	}
 
-	args = append(args, rs.Source)
-	args = append(args, fmt.Sprintf("%s@%s:%s", rs.User, rs.host, rs.Target))
+	srcs, err := filepath.Glob(path.Join(root, rs.Source))
+	if err != nil {
+		args = append(args, rs.Source)
+	} else {
```

Comment:
maybe do `if err != nil || len(srcs) == 0` ... just so we can get rid of some of the nesting below